src/main/resources/jenkins/plugins/publish_over_cifs/Messages.properties

```diff
@@ -37,6 +37,7 @@ console.config.noWins=Removing WINS from name resolution
 console.config.wins=Setting WINS server [{0}]
 console.config.timout=Setting response timeout [{0}]
 console.config.soTimeout=Setting socket timeout [{0}]
+console.config.bufferSize=Setting buffer size [{0}]
```

Comment:
I don't see this actually being used in the code, is it used somewhere?

Reply:
I completely forgot to use it, thank you for that information. Btw. I've added the usage of that command in the newest change => f2bcff9b4c045b9f9cfd0c29a3f6ddbe955078cb